webapp/mailing/models.py

```diff
@@ -6,6 +6,7 @@
 class MailTemplate(TimeStampedModel):
     title = models.CharField(max_length=100)
     subject = models.CharField(max_length=100, default="VOKO Utrecht - ")
+    from_email = models.CharField(max_length=100, default="", blank=True)
```

Comment:
Can we make it nullable with default=None?

Reply:
Definitely, I didn't realize that was better.

Reply:
Please replace `blank` with `null` and remove the `default` (the default will be None/null)

Reply:
> Or maybe we should consider changing the permissions so that transport coordinators can change the mail templates? (`mailing | mail template | Can change mail template`)

It's just that they will be able to change *all* templates and I'm afraid people will make mistakes. So I prefer the first method :)

Reply:
> Please replace `blank` with `null` and remove the `default` (the default will be None/null)

But:

> Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;

https://docs.djangoproject.com/en/1.11/ref/models/fields/#null

Reply:
Oops, you're right! Looks like it was perfect the way it was. 
You might have to change the default back to empty string, not sure if this works...

Reply:
I've reverted it back to an empty string.